integration-tests/discovery-servicecomb-tests/account-provider/src/main/resources/bootstrap.yaml

```diff
@@ -59,6 +59,7 @@ spring:
 
           securityPolicyEnabled: true
           acls:
+            tokenCheckEnabled: true
```

Comment:
add as a global configuration not only for acls.

Reply:
fixed